config/crds/starlingx_v1_host.yaml

```diff
@@ -63,6 +63,15 @@ spec:
                     address:
                       description: Address defines the board management IP address.
                       type: string
+                    type:
```

Comment:
I will assume that since this passed your testing,  the field name "type"  does not conflict with the "type" reserved field